### PR TITLE
fix(connectors): lnbits incoming payments shows outgoing ones as well

### LINKED ISSUE #1631: Sent transaction appear on received tab as +-
### Describe the bug

I did a lightningaddress transaction using Alby (The account connected is Lnbits from Umbrel), and the transaction appear on received tabs with +-
obs. Appear on sent tab too correctly

### Screenshots of the changes [optional]

photo_2022-10-18_14-25-12

### To Reproduce

1- Connect a Lnbits wallet on Alby
2- Make a lightning transaction

### Expected behavior

Show the transaction only on Sent tab

### Alby information

- Alby Version: 1.17
- Alby installed through [e.g. installed through the browser stores, installed manually]
- Wallet connected with Alby: Lnbits from Umbrel

### Are you working on this issue?

No

### PR DESCRIPTION
**Pull request by escapedcat**
### Describe the changes you have made in this PR

lnbits `payments` api shows outgoing and incoming payments. Negative ones are outgoing and need to be filtered out for our incoming txs.

### Link this PR to an issue [optional]

Fixes #1631

### Type of change

- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes [optional]

image

### How has this been tested?

Manually by sendign sats back and forth

### CODE REVIEW

File: src/extension/background-script/connectors/lnbits.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ class LnBits implements Connector {
         data: {
           checking_id: string;
           pending: boolean;
-          amount: string;
+          amount: number;
```
Comment (escapedcat) [inline src/extension/background-script/connectors/lnbits.ts:99]: It's actually a number:
```
  /*
  LNBits Swagger Docs: https://legend.lnbits.org/devs/swagger.html#/default/api_payments_api_v1_payments_get
  Sample Response:
    [
      {
        "checking_id": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
        "pending": false,
        "amount": 2000,
```

Comment (escapedcat) [reply src/extension/background-script/connectors/lnbits.ts:99]: image

**Comment by escapedcat** [comment]
> did you test it (potentially with different lnbits installations)?

Isn't the API behaving the same among lnbits installations? Where else could I test it?

> is the ordering correct (maybe also add the sorting there?)?

Sorting looks good, no? See screenshot above.

**Comment by bumi** [comment]
if you tested it and everything works, then merge it. 
I'd still add the sorting (which you added also for lnd)?

**Comment by escapedcat** [comment]
> if you tested it and everything works, then merge it. 

Tested with our lnbits account form the e2e tests. 

> I'd still add the sorting (which you added also for lnd)?

The sort takes care of an API flaw which doesn't exist in the lnbits API. So I would not add it.

**Review by reneaaron — APPROVED** [review]

**Pull request merged**